dwbb/src/interface/DW_ram_2r_2w_s_dff.scala

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 Jiuyang Liu <liu@jiuyang.me>
+package org.chipsalliance.dwbb.interface.DW_ram_2r_2w_s_dff
+
+import chisel3._
+import chisel3.experimental.SerializableModuleParameter
+import upickle.default
+
+object Parameter {
+  implicit def rw: default.ReadWriter[Parameter] =
+    upickle.default.macroRW[Parameter]
+}
+
+case class Parameter(
+    width: Int = 8,
+    addrWidth: Int = 3,
+    rstMode: Boolean = false
```

Comment:
what is rstMode?

Reply:
It would be clearer and simpler to be a `Boolean` type.